<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>700</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab {
  color: #171717;
  background: transparent;
  font-weight: bold;
  letter-spacing: 1px;
  width: 200px;
  height: 40px;
  padding-left: 25px;
  padding-right: 25px;
}

QTabBar::tab:hover {
  border-bottom: 3px solid #bbbbff;
}

QTabBar::tab:selected {
  border-bottom: 3px solid #2A50D3;
}
</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>KODIEREN</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>341</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>PT Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #777777;</string>
         </property>
         <property name="text">
          <string>Zu kodierender Text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="encode_text_field">
         <property name="font">
          <font>
           <family>PT Sans</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #eaeaea;
/* border: 1px solid #dadada; */
border-radius: 5px;
padding: 8px 14px 8px 14px;</string>
         </property>
         <property name="text">
          <string>Gosig Golden ist süß!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="encode_image">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="generate_qr_button">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>340</width>
        <height>46</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0;
border-radius: 5px;
background: #2A50D3;
color: white;
letter-spacing: 1px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>QR-CODE GENERIEREN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_qr_button">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>580</y>
        <width>340</width>
        <height>46</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0;
border-radius: 5px;
background: #2A50D3;
color: white;
letter-spacing: 1px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>SPEICHERN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>339</width>
        <height>17</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #777777;</string>
      </property>
      <property name="text">
       <string>Generierter QR-Code</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>400</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top: 1px solid #dadada;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>DEKODIEREN</string>
     </attribute>
     <widget class="QPushButton" name="upload_qr_button">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>370</y>
        <width>340</width>
        <height>46</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0;
border-radius: 5px;
background: #2A50D3;
color: white;
letter-spacing: 1px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>HOCHLADEN</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>450</y>
        <width>341</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>PT Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #777777;</string>
         </property>
         <property name="text">
          <string>Dekodierter Text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="decode_text_field">
         <property name="font">
          <font>
           <family>PT Sans</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #eaeaea;
/* border: 1px solid #dadada; */
border-radius: 5px;
padding: 8px 14px 8px 14px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="decode_image">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>430</y>
        <width>400</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top: 1px solid #dadada;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>51</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>10</y>
      <width>102</width>
      <height>33</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>51</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
